com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -647,31 +642,44 @@ public SocketTasks StartHost(Vector3? position = null, Quaternion? rotation = nu
 
         public void SetSingleton()
         {
-            Singleton = this;
-
-            if (OnSingletonReady != null)
-                OnSingletonReady();
+            OnSingletonReady?.Invoke();
```

Comment:
this function and `OnSingletonReady` event should be gone, probably.

Reply:
yes, removed it.